docs/dc-apps-performance-toolkit-user-guide-confluence.md

```diff
@@ -60,10 +60,11 @@ All important parameters are listed and described in this section. For all other
 | --------- | ----------------- |
 | Collaborative editing mode | synchrony-local |
 | Confluence Version | 6.13.8 |
+| Confluence Version | 7.0.4  |
```

Comment:
`| Confluence Version | 6.13.8 or 7.0.4 |`

Reply:
Done, thx for showing this issue!